### PR TITLE
#155 alignment is resolved

### PR DESCRIPTION
**Pull request by gunjanmimo**
Signed-off-by: gunjanmimo <gunjan.mtbpaul@gmail.com>

### CODE REVIEW

File: _includes/getting-started.html [hunk 1]
```diff
@@ -2,10 +2,10 @@
     <div style="margin-top: auto;" id="box"> 
       <h1 style="text-align: left;margin-left: 80px;"><strong style="font-size: 90px;">Get Started</strong><br>
         with Meshery
-      <p style="font-size: 20px;">Choose a platform and see how easy it is to get up and running with a service mesh.</p></h1>
+      <p style="font-size: 20px;">Choose a platform and see how easy<br> it is to get up and running with a service mesh.</p></h1>
     </div>
   </div> 
-<div class="container flex" id="containers" style="margin-left: 680px;margin-top: -250px;">
+<div class="container flex" id="containers" style="margin-left: 680px;margin-top: -210px;">
```
Comment (leecalcote) [inline _includes/getting-started.html:8]: You'll want to work toward a more dynamic approach than specifying in pixels. Resources:

- https://webdesign.tutsplus.com/tutorials/a-comprehensive-guide-to-flexbox-sizing--cms-31948
- https://css-tricks.com/snippets/css/a-guide-to-flexbox/

Comment (gunjanmimo) [reply _includes/getting-started.html:10]: @leecalcote sir, now I have solved this issue

**Comment by gunjanmimo** [comment]
> image
> @gunjanmimo There is still issue on smaller screens

hi @shivaylamba, now this issue is solved. I have no idea how to show those command in mobile view, do have any idea to solve this issue?

**Comment by shivaylamba** [comment]
@gunjanmimo Could you share a live preview or screenshot

**Comment by gunjanmimo** [comment]
Screenshot from 2020-04-26 21-18-34
@shivaylamba Here is my last update

**Comment by leecalcote** [comment]
@rubenlorenzo or @Yashank18 might have some thoughts / feedback to offer here.

**Comment by Yashank18** [comment]
@gunjanmimo  @leecalcote  I just saw the changes and I think this may fix the side alignment issues but there will be an issue when the user will select Kubernetes, the "innerdiv" data come over "Login" heading. We need to set up the alignment of each div accordingly.

**Comment by gunjanmimo** [comment]
@Yashank18 may you check now? I have solved the issue?

File: Gemfile [hunk 2]
```diff
@@ -1,6 +1,6 @@
 source 'https://rubygems.org'
 
-ruby '2.6.5'
+ruby '2.7.0'
```
Comment (leecalcote) [inline Gemfile:3]: I think we're probably ready to upgrade to use of 2.7.0 as a stable Ruby release across each of our Jekyll instances.

In order to do so completely, we'll want to update the Netlify config simultaneously. // @kushthedude

**Comment by Yashank18** [comment]
@gunjanmimo  changes looks fine to me, but can you share some screenshots in tab view and mobile view?

**Comment by leecalcote** [comment]
The Netlify builds are failing due to a mismatch of Ruby versions configured. We'll either need to stay at 2.6.5 here or upgrade the Netlify config to 2.7.0

File: Gemfile (hunk 2 above)
Comment (kushthedude) [reply Gemfile:3]: Let's move to manual `netlify.toml` rather than using their UI Panel.

**Comment by kushthedude** [comment]
Adding netlify bot's comments & updating the ruby version for this PRs sake.

**Comment by kushthedude** [comment]
Now try pushing a commit.

**Comment by gunjanmimo** [comment]
@kushthedude may you check now?
and @leecalcote sir, I have added a small animation in getting started section. may you check the change?

File: _includes/getting-started.html [hunk 3]
```diff
@@ -1,106 +1,124 @@
...
-      <h1 style="text-align: left;margin-left: 80px;"><strong style="font-size: 90px;">Get Started</strong><br>
-        with Meshery
-      <p style="font-size: 20px;">Choose a platform and see how easy it is to get up and running with a service mesh.</p></h1>
-    </div>
-  </div> 
-<div class="container flex" id="containers" style="margin-left: 680px;margin-top: -250px;">
-<div class="text editable code">
+<div id="getting-started">
+  <div style="margin-top: auto;" id="box">
+    <h1 style="text-align: left;margin-left: 80px;"><strong style="font-size: 90px;">Get Started</strong><br>
```
Comment (kushthedude) [inline _includes/getting-started.html:3]: Shouldn't this be Getting Started ?

Comment (kushthedude) [inline _includes/getting-started.html:3]: Screenshot 2020-04-28 at 9 12 33 PM
Can this be fixed ?

File: _includes/getting-started.html [hunk 4]
```diff
@@ -1,106 +1,124 @@
...
-        <h2><strong>Install and Start</strong></h2>
-        <h1 style="font-size: 80px;opacity: 0.2;margin-top: -95px;margin-left: -55px;"><strong>01</strong></h1>
+      <h2><strong>Install and Start</strong></h2>
+      <h1 style="font-size: 80px;opacity: 0.2;margin-top: -95px;margin-left: -55px;"><strong>01</strong></h1>
 
-       <!-----------try---------->
-       <label for="start-helm-v2" data-tooltip="Bash User"><img id="bounce" src="https://upload.wikimedia.org/wikipedia/commons/thumb/4/4b/Bash_Logo_Colored.svg/1024px-Bash_Logo_Colored.svg.png" alt="" style="height: 50px;width: 50px;cursor:pointer;"></label>
+
+      <!-- <label for="start-helm-v2" data-tooltip="Bash User"><img id="bounce" src="https://upload.wikimedia.org/wikipedia/commons/thumb/4/4b/Bash_Logo_Colored.svg/1024px-Bash_Logo_Colored.svg.png" alt="" style="height: 50px;width: 50px;cursor:pointer;"></label>
         <label for="start-brew" data-tooltip="Brew User"><img id="bounce" src="https://brew.sh/assets/img/homebrew-256x256.png" alt="" style="height: 50px;width: 50px;margin-left: 60px;top: -81px;position: relative;cursor:pointer"></label>
```
Comment (kushthedude) [inline _includes/getting-started.html:18]: Can we remove the commented code here ?

File: _includes/getting-started.html [hunk 5]
```diff
@@ -109,126 +127,123 @@ <h4 style="font-size: 1.5em;">Using Helm</h4>
...
-    <h1 style="font-size: 80px;opacity: 0.2;margin-top: -95px;margin-left: -55px;"><strong>02</strong></h1>
-    <p style="margin-left:1em;">Access Meshery in your browser if you are using Kubernetes at <a href="http://127.0.0.1:9081" style="color: rgb(85, 217, 240);">http://127.0.0.1:9081</a>.</p>
-    </div>
-    
-    <!-- <h3 style="color:#111111;">Step 3: Configure and Enjoy Meshery</h3>
+        <hr>
+        <div class="login">
+          <h2 style="margin-top: 1em;"><strong>Login</strong></h2>
+          <h1 style="font-size: 80px;opacity: 0.2;margin-top: -95px;margin-left: -55px;"><strong>02</strong></h1>
+          <p style="margin-left:1em;">Access Meshery in your browser if you are using Kubernetes at <a
```
Comment (kushthedude) [inline _includes/getting-started.html:237]: Screenshot 2020-04-28 at 9 13 39 PM

Was this included in this part of the issue or is it other? I am not sure if this is also an alignment issue.

File: _includes/getting-started.html (hunk 3 above)
Comment (gunjanmimo) [reply _includes/getting-started.html]: may you check again? I think now this issue is solved.

**Comment by kushthedude** [comment]
The animation seems quite weird in small screens & large screens. @leecalcote What do you say ?

**Comment by gunjanmimo** [comment]
@leecalcote sir, this pr is ready to merge.

**Comment by leecalcote** [comment]
Thank you, @gunjanmimo! @rubenlorenzo will you review?

**Comment by rubenlorenzo** [comment]
@leecalcote How does the review work? I haven't done any.

**Comment by rubenlorenzo** [comment]
@leecalcote I have read this document [Reviewing proposed changes in a pull request - GitHub Help](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/reviewing-proposed-changes-in-a-pull-request). I can look at it tomorrow more closely.

File: _includes/getting-started.html [hunk 6]
```diff
@@ -1,234 +1,256 @@
...
+            <div class="hidediv">
+              <div class="innerdiv">
+                <p>Download and unzip <span style="color: rgb(117, 250, 255);">mesheryctl</span> from the <a
+                    href="https://github.com/layer5io/meshery/releases/tag/v0.3.9">Meshery releases</a> page. Add <span
+                    style="color: rgb(117, 250, 255);">mesheryctl</span> to your PATH for ease of use. Then, execute:
+                </p>
+                <div id="start" class="highlight-code highlight-source-shell" style="margin-left:1.8em;">
+                  <div style="width:auto; height:30px; padding:0px; position:relative;">
+                    <pre><code>   $ ./mesheryctl start
+                </code></pre>
```
Comment (rubenlorenzo) [inline _includes/getting-started.html:220]:
```suggestion
                    <pre>
                         <code>   $ ./mesheryctl start</code>
                    </pre>
```

Comment (rubenlorenzo) [inline _includes/getting-started.html:220]:
```suggestion
                    <pre>
                         <code>   $ ./mesheryctl start </code>
                    </pre>
```

File: _includes/getting-started.html [hunk 7]
```diff
@@ -1,234 +1,256 @@
...
-                data-clipboard-target="#start"
-                data-clipboard-text="./mesheryctl start">
-                <img src="/images/buttons/clippy.svg" style="width:20px; margin: 0; position: absolute; top: 50%; left: 54%; transform: translate(-50%, -50%);" />
-              </a>
+          <hr>
+          <div class="login">
+            <h2 style="margin-top: 1em;"><strong>Login</strong></h2>
+            <h1 style="font-size: 80px;opacity: 0.2;margin-top: -95px;margin-left: -55px;"><strong>02</strong></h1>
+            <p style="margin-left:1em;">Access Meshery in your browser if you are using Kubernetes at <a
+                href="http://127.0.0.1:9081" style="color: rgb(85, 217, 240);">http://127.0.0.1:9081</a>.</p>
```
Comment (rubenlorenzo) [inline _includes/getting-started.html:240]:
```suggestion
            <p style="margin-left:1em;">
                 Access Meshery in your browser if you are using Kubernetes at <a href="http://127.0.0.1:9081" style="color: rgb(85, 217, 240);">http://127.0.0.1:9081</a>.
            </p>
```

File: _includes/getting-started.html [hunk 8]
```diff
@@ -1,234 +1,256 @@
...
-              </a>
+
+          <!-- Docker -->
+          <div class="toggle">
+            <!-- <h3 style="margin-left:1em;">Windows user</h3 style="margin-left:1.5em;"> -->
+            <label for="windows" style="margin-left:1.8em;display: inline;	font-size: 1.38em;
+        color: #666;
+        font-weight: 300;
+        line-height: 1.4em;">
+              <a></a>.</label>
```
Comment (rubenlorenzo) [inline _includes/getting-started.html:209]:
```suggestion
            <label for="windows" 
                 style="margin-left:1.8em;display: inline; font-size: 1.38em;
                  color: #666; font-weight: 300; line-height: 1.4em;">
                       <a></a>.
              </label>
```

File: _includes/getting-started.html [hunk 9]
```diff
@@ -1,234 +1,256 @@
...
+          <div> <label for="start-brew" data-tooltip="Brew User"><img id="bounce"
+                src="https://brew.sh/assets/img/homebrew-256x256.png" alt=""></label>
+          </div>
+          <div><label for="k8s-instructions" data-tooltip="Kubernetes User"><img id="bounce"
+                src="https://upload.wikimedia.org/wikipedia/commons/thumb/3/39/Kubernetes_logo_without_workmark.svg/1200px-Kubernetes_logo_without_workmark.svg.png"
+                alt=""></label>
+          </div>
+          <div><label for="windows" data-tooltip="Windows User"><img id="bounce"
+                src="https://pngimg.com/uploads/windows_logos/windows_logos_PNG25.png" alt=""></label>
+          </div>
```
Comment (rubenlorenzo) [inline _includes/getting-started.html:40]:
```suggestion
          <div>
               <label for="start-helm-v2" data-tooltip="Bash User">
                    <img id="bounce"
                         src="https://upload.wikimedia.org/wikipedia/commons/thumb/4/4b/Bash_Logo_Colored.svg/1024px-                        Bash_Logo_Colored.svg.png"
                          alt="">
               </label>
          </div>
          <div>
               <label for="start-brew" data-tooltip="Brew User">
                    <img id="bounce"
                         src="https://brew.sh/assets/img/homebrew-256x256.png" alt="">
                 </label>
          </div>
          <div>
               <label for="k8s-instructions" data-tooltip="Kubernetes User">
                    <img id="bounce"
                         src="https://upload.wikimedia.org/wikipedia/commons/thumb/3/39/Kubernetes_logo_without_workmark.svg/1200px-Kubernetes_logo_without_workmark.svg.png"
                          alt="">
               </label>
          </div>
          <div>
               <label for="windows" data-tooltip="Windows User">
                     <img id="bounce"
                          src="https://pngimg.com/uploads/windows_logos/windows_logos_PNG25.png" alt="">
                </label>
          </div>
```

**Comment by rubenlorenzo** [comment]
Hi @gunjanmimo, my comments were for tabulation. The rest, it's fine for me. I see that it is linked to my issue #155. I don't understand it altogether. Do I have to do something more?

**Comment by gunjanmimo** [comment]
@leecalcote sir, I am facing dco issue. How to solve this issue?

**Comment by shivaylamba** [comment]
@gunjanmimo You need to rebase your existing commit, and use the -s command to sign the commit. And then force push.

**Comment by Nikhil-Ladha** [comment]
@gunjanmimo I saw the code for the getting started section and it seems like the code is completely messed up there is usage of margin properties in a lot of place which can lead to issues in case of responsiveness. My suggestion would be go through the code at that area and rectify it one by one. Try using better css properties for fixing responsiveness issue and it will be good.
And also, check if the order in which the div are if that's correct, because it seemed like a issue there which may cause other errors.

**Comment by gunjanmimo** [comment]
@Nikhil-Ladha  my changes replaced with the current code mistakenly. so I am working on this to make it better

File: _includes/getting-started.html [hunk 10]
```diff
@@ -1,106 +1,117 @@
...
-                <div style="width:auto; height:30px; padding:0px; position:relative;border-radius: 5px;">
-                  <pre><code>   $ curl -L https://git.io/meshery | bash -
+          <a></a>.</label>
+        <input type="checkbox" class="k8s-instructions" checked name="one" id="start-helm-v2">
+        <div class="hidediv">
+          <div class="innerdiv">
+            <p>Install on Mac or Linux using Docker:</p>
+
+            <div id="start" class="highlight-code highlight-source-shell" style="margin-left:1.8em;">
+              <div style="width:auto; height:30px; padding:0px; position:relative;border-radius: 5px;">
```
Comment (Vineet-Sharma29) [inline _includes/getting-started.html:51]: @gunjanmimo Can you check for margins over here? Although you will have to fix margin and padding issue in parent elements as well. It is causing following issues:-

Desktop view:-
Content is overflowing in x direction
Screenshot from 2020-06-16 12-11-03

File: _includes/getting-started.html [hunk 11]
```diff
@@ -109,126 +120,118 @@ <h4 style="font-size: 1.5em;">Using Helm</h4>
...
-    <p>- Configure Meshery with references to your deployed adapters, using "meshery-&lt;service-mesh-name&gt;:&lt;port&gt;" (e.g. <code>meshery-istio:10000</code> or <code>meshery-linkerd:10001</code>)
-      in the <i>Mesh Adapter Location</i> field.</p> -->
+        <hr>
+        <div class="login">
+          <h2 style="margin-top: 1em;"><strong>Login</strong></h2>
+          <h1 style="font-size: 80px;opacity: 0.2;margin-top: -95px;margin-left: -55px;"><strong>02</strong></h1>
+          <p id="breakWords" style="margin-left:1em;">Access Meshery in your browser if you are using Kubernetes at <a
+              href="http://127.0.0.1:9081" style="color: rgb(85, 217, 240);">http://127.0.0.1:9081</a>.</p>
+        </div>
+      </div>
```
Comment (Vineet-Sharma29) [inline _includes/getting-started.html:232]: @gunjanmimo There seems to be margin and padding issue over here. To fix margin and padding in parent elements also needs to be fixed

Desktop View 
Text in login section  is overflowing in x-direction
Screenshot from 2020-06-16 11-45-22

File: _sass/layout.scss [hunk 12]
```diff
@@ -250,177 +250,140 @@ table, th, td {
...
-	
 
-  }
+}
 
-  @media screen and (max-width: 500px){
-	#box{
-	  width: 100%;
-	  font-size: small;
+@media screen and (max-width: 1200px) {
```
Comment (Vineet-Sharma29) [inline _sass/layout.scss:321]: @gunjanmimo  Can you add media queries for mobile and tablet view as well? I think that would solve following errors:-

Mobile View:-

Screenshot from 2020-06-16 11-48-01


Tablet View:-

Screenshot from 2020-06-16 11-50-22

**Comment by leecalcote** [comment]
> There are still issues in mobile, tablet and desktop view. I think a revised margin and padding from parent to child elements would solve the issues. [This](https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/The_box_model) article on box-model for CSS would help.

Thank you, @Vineet-Sharma29. @gunjanmimo will you be giving this another attempt?

File: _includes/getting-started.html [hunk 13]
```diff
@@ -1,106 +1,118 @@
...
-      <h1 style="text-align: left;margin-left: 80px;"><strong style="font-size: 90px;">Get Started</strong><br>
-        with Meshery
-      <p style="font-size: 20px;">Choose a platform and see how easy it is to get up and running with a service mesh.</p></h1>
-    </div>
-  </div> 
-<div class="container flex" id="containers" style="margin-left: 680px;margin-top: -250px;">
-<div class="text editable code">
+<div class="container flex">
+
+  <div class="anchor" style="display:flexbox;" id="box"><a name="getting-started"></a>
```
Comment (leecalcote) [inline _includes/getting-started.html:3]: I added the `class="anchor"` tag back, so that the nav menu doesn't overlap the section heading.

**Comment by Nikhil-Ladha** [comment]
The PR #237 references this issue.

**Comment by leecalcote** [comment]
See #237

**Pull request closed without merging**